src/luxonis_ml/utils/config.py

```diff
@@ -70,7 +93,7 @@ def save_data(self, path: str) -> None:
             path (str): Path to output yaml file
         """
         with open(path, "w+") as f:
-            yaml.dump(self._data.model_dump(), f, default_flow_style=False)
+            yaml.dump(self.model_dump(), f, default_flow_style=False)
 
     def get(self, key_merged: str, default: Any = None) -> Any:
```

Comment:
Is this still needed or should we remove it? Maybe a point for keeping it would be still support "safe" get() with default value if it doesn't exist (similar as python dict).

Reply:
Yeah, the `get` is still very useful (for example `MetadataLogger` depends on it).